cas/service.c

```diff
@@ -1193,6 +1193,7 @@ static int cas_fom_tick(struct m0_fom *fom0)
 		M0_LOG(M0_DEBUG, "Got CAS with txid: " DTID0_F,
 		       DTID0_P(&op->cg_txd.dtd_id));
 	}
+
```

Comment:
Here it was a line with space. While this change is needless, removing it is correct.